targets/XPlatCppSdk/source/com.playfab.xplatcppsdk.v141.autopkg.ejs

```diff
@@ -40,7 +40,7 @@ nuget {
         };
 
         nestedInclude: {
-            #destination = ${d_include}\playfab;
+            #destination = ${d_include}playfab;
```

Comment:
${d_include} ends with a \
${d_include}\playfab creates an unnamed folder. Removed the \